validator/params.json.template

```diff
@@ -1,10 +1,13 @@
 {
   "validation": {
-  "data_source": "fb_survey",
-  "start_date": "2020-06-13",
-  "end_date": "2020-06-19",
-  "ref_window_size": 7,
-  "data_folder": "C:/Covidcast/covidcast-indicators/validator/data",
-  "minimum_sample_size": 100
+    "data_source": "usa-facts",
+    "start_date": "2020-09-05",
+    "end_date": "2020-09-08",
```

Comment:
We discussed this adequately this morning so just noting here for posterity: prefer to configure end_date and span_length (doctor-visits produces updates on ~6 weeks of data at each issue), with special end_date value like "latest" or "new" to indicate should just use date.today().

Reply:
Does it make sense to check the not-yet-finalized (i.e. still in the *x* week update window) data? Presumably, it will raise a lot of errors, especially the most-recent data, for being very different than historical values. And we'll also end up checking those dates multiple times, as they move from less-finalized to more-finalized.

Reply:
I see what you mean, but if a 100x magnitude change happens during the backfill window, we'll want to know about it as soon as possible. One thing we can do when it comes time to install this in each indicator is _run_ the validator but not _believe_ the validator for the first week or so, just to see what kinds of false positives are raised. Then we can revise accordingly.

Reply:
> Does it make sense to check the not-yet-finalized (i.e. still in the x week update window) data? Presumably, it will raise a lot of errors

Sounds like this is a non-issue. I was imagining most-recent data starting off with only, say 5% of the data ultimately available for that day, but you said above it's more like 70%. So high missingness, low sample size leading to biased values, etc are unlikely.